TFNetworkLayer.py

```diff
@@ -5419,7 +5419,7 @@ def reduce_to_batch(self, loss, normalize):
       # We expect to get (batch*time) or (time*batch) in the first dimension of the loss and the output.
       loss = self._reduce_to_batch_time_with_mask(loss, normalize=normalize)  # (batch*time,) or (time*batch,)
       loss.set_shape((None,))
-      loss = tf.reshape(loss, tf.shape(self.output.get_sequence_mask()))  # (batch,time) or (time,batch)
+      loss = tf.reshape(loss, tf.shape(self.loss.output.placeholder)[:2])  # (batch,time) or (time,batch)
```

Comment:
Why `self.loss.output` and not `self.output`?

Reply:
that was your suggestion :)
Is fixed now